_component_design/actions-menu.md

```diff
@@ -1,17 +1,100 @@
 ---
 title: Actions Menu
-parent: Editing
-secondary: Inputs and Controls
+parent: Inputs and Controls
+secondary: Editing
 layout: component
 category: Components
 usage: >
   The actions menu is used anytime a group of actions can be performed on a specific object on the current page or the current page. By grouping these actions within the actions menu, it allows users to have a single location to trigger multiple types of actions.
-preview-image: components/preview-help.svg
-status: unknown
-resource: false
-last-modified: 2017-08-17
+preview-image: preview-images/actions-menu.svg
+status: stable
+resource: true
+last-modified: 2017-11-03
 ---
 
 {% include toc.html %}
 
-<!-- unknown -->
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Actions Menus are used when to consolidate multiple related action links into a single menu. 
```

Comment:
Use actions menus to consolidate multiple related-action links into a single menu.